test/support/encoded_schema.json

```diff
@@ -0,0 +1,472 @@
+{
+  "openapi": "3.0.0",
```

Comment:
There's a few errors in this file. Try pasting into https://editor.swagger.io to see them all.

Reply:
Good idea!